README.md

```diff
@@ -20,3 +20,4 @@ Each example contains a README.md with an explanation about the solution and it'
 | [`slack-code-runner-bot`](slack-code-runner-bot) <br/> Run JavaScript code snippets directly from your own Slack | nodeJS |
 | [`trello-auto-updates`](trello-auto-updates) <br/> Implement Trello webhooks to extend a dashboard behavior | nodeJS |
 | [`whiteboard`](whiteboard) <br/> Synchronized web whiteboard based on functions, Redis and websockets | nodeJS |
+| [`python-crud-google-spreadsheet`](python-crud-google-spreadsheet) <br/> Google Spreadsheet CRUD with Functions | python3 |
```

Comment:
Why is functions capitalized.

Also it's "Google Sheets" not "Google Spreadsheet"